salt/modules/debbuild.py

```diff
@@ -170,8 +175,8 @@ def _get_repo_dists_env(env):
         'ORIGIN': ('O', 'Origin', 'SaltStack'),
         'LABEL': ('O', 'Label', 'salt_debian'),
         'SUITE': ('O', 'Suite', 'stable'),
-        'VERSION': ('O', 'Version', '8.1'),
-        'CODENAME': ('M', 'Codename', 'jessie'),
+        'VERSION': ('O', 'Version', '9.0'),
+        'CODENAME': ('M', 'Codename', 'stretch'),
```

Comment:
When updating for better handling on Debian 9 of gpg2, I noticed the default values, and corrected them for Debian 9 since that is now stable.  Normally when building, the state files set theses values, but I thought I should change the default values when I came across them.